src/services/inventory/cloud-service/modules/cloud-service-list/CloudServiceListCard.vue

```diff
@@ -3,33 +3,39 @@
-                            class="icon"
-                />
-                <span class="provider">{{ providers[item.provider] ? providers[item.provider].label : item.provider }}</span>
-            </div>
-            <div class="service-group-wrapper">
-                <p-lazy-img width="1.5rem" height="1.5rem"
-                            :src="assetUrlConverter(item.icon) || (providers[item.provider] ? providers[item.provider].icon : '')"
-                            error-icon="ic_provider_other"
-                            :alt="item.name"
-                            class="icon"
-                />
-                <span class="service-group">{{ item.cloud_service_group }}</span>
+            <div class="card-title-wrapper">
+                <div class="provider-title-wrapper">
+                    <span class="provider">{{ providers[item.provider] ? providers[item.provider].label : item.provider }}</span>
+                </div>
+                <div class="service-group-wrapper">
+                    <p-lazy-img width="1.25rem" height="1.25rem"
+                                :src="assetUrlConverter(item.icon) || (providers[item.provider] ? providers[item.provider].icon : '')"
+                                error-icon="ic_provider_other"
+                                :alt="item.name"
+                                class="icon"
+                    />
+                    <span class="service-group">{{ item.cloud_service_group }}</span>
+                </div>
             </div>
+            <p-divider />
             <div class="service-type-list">
-                <router-link v-for="(cloudServiceType, idx) in item.resources"
-                             :key="`${cloudServiceType}-${idx}`"
-                             :to="getCloudServiceDetailLink({ ...item, cloudServiceTypeName: cloudServiceType.cloud_service_type })"
-                             class="service-type-item"
+                <template
+                    v-for="(cloudServiceType, idx) in item.resources"
                 >
-                    <span class="service-type-name">{{ cloudServiceType.cloud_service_type }}</span>
-                    <span class="service-type-count">{{ cloudServiceType.count }}</span>
-                </router-link>
+                    <router-link
+                        :key="`${cloudServiceType}-${idx}`"
+                        :to="getCloudServiceDetailLink({ ...item, cloudServiceTypeName: cloudServiceType.cloud_service_type })"
+                        class="service-type-item"
+                    >
+                        <span class="service-type-name">{{ cloudServiceType.cloud_service_type }} </span>
+                        <span class="service-type-count">{{ cloudServiceType.count }}</span>
+                    </router-link>
+                    <p-divider
+                        v-if="item.resources.length > 1 && idx % 2 === 0"
+                        :key="idx"
+                        vertical="true"
```

Comment:
```suggestion
                        :vertical="true"
```

or

```suggestion
                        vertical
```

---

true를 주고싶으면 간소하게 2번째처럼 바인딩할 수 있습니다. (functional 컴포넌트의 경우 제대로 동작 안함)
`v-bind (:bind)` 방식으로 바인딩하지 않으면 기본적으로 string으로 인식하므로 반드시 첫번쨰처럼 바인딩해주어야 합니다.